tests/devices/test_default_mixed.py

```diff
@@ -253,6 +258,14 @@ class TestKrausOps:
         (PauliX, np.array([[0, 1], [1, 0]])),
         (Hadamard, np.array([[INV_SQRT2, INV_SQRT2], [INV_SQRT2, -INV_SQRT2]])),
         (CNOT, np.array([[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, 1], [0, 0, 1, 0]])),
+        (
+            PauliError("X", 0.5, wires=0),
+            [np.sqrt(0.5) * np.eye(2), np.sqrt(0.5) * np.array([[0, 1], [1, 0]])],
+        ),
+        (
+            PauliError("Y", 0.5, wires=0),
+            [np.sqrt(0.5) * np.eye(2), np.sqrt(0.5) * np.array([[0, -1j], [1j, 0]])],
```

Comment:
This way we can make sure we assign the correct probabilities to the correct error.
```suggestion
            PauliError("Y", 0.3, wires=0),
            [np.sqrt(0.7) * np.eye(2), np.sqrt(0.3) * np.array([[0, -1j], [1j, 0]])],
```